SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -1372,7 +1372,9 @@ public void sessionEnd() {
     @Override
     public void event(String name) {
         try {
-            _event(name);
+            if(isValidEventName(name)) {
+                _event(name);
+            }
```

Comment:
Can we add a logger statement here to notify the user that they're event couldn't be sent due to it starting with Swrve?